dashboard/src/components/DeploymentForm/DeploymentForm.test.tsx

```diff
@@ -95,26 +106,23 @@ it("renders the full DeploymentForm", () => {
   const wrapper = shallow(
     <DeploymentForm {...defaultProps} selected={{ versions, version: versions[0] }} />,
   );
-  // Force static name
-  wrapper.setState({ releaseName: "bar" });
   expect(wrapper).toMatchSnapshot();
 });
 
 it("renders a release name by default", () => {
```

Comment:
Maybe update the description of the test case to mention that it relies in Monikers output